css/arrow.css

```
@@ -86,3 +86,15 @@
 .flicking-arrow-disabled::after {
   background-color: #e6e6e6;
 }
+
+.flicking-arrow-prev.thin::before,
+.flicking-arrow-prev.thin::after,
+.flicking-arrow-next.thin::before,
+.flicking-arrow-next.thin::after{
+  height: 3px;
+}
+
+.flicking-arrow-prev.thin::after,
+.flicking-arrow-next.thin::after{
+  top : calc(50% - 2px);
```

Please remove the space between `top` and `:`

```suggestion
  top: calc(50% - 2px);
```